File: problems/problems_2255/Solution.java
package problems.problems_2255;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int countPrefixes(String[] words, String s) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        String[] words = jsonArrayToStringArray(inputJsonValues[0]);
		String s = jsonStringToString(inputJsonValues[1]);
        return JSON.toJSON(countPrefixes(words, s));
    }
}

File: problems/problems_2255/solution.py
import solution
from typing import *


class Solution(solution.Solution):
    def solve(self, test_input=None):
        return self.countPrefixes(*test_input)

    def countPrefixes(self, words: List[str], s: str) -> int:
        return sum(s.startswith(v) for v in words)

